scripts/konflux/fetch-scanner-data.sh

```diff
@@ -16,14 +16,28 @@ shift
 
 blobs=( "$@" )
 
+# Ensure that we download scanner data for a release if this is a tagged build.
+
+# First, try take git tag if it's a tagged commit.
+tag="$(git tag --points-at)"
+if [[ -z "${tag}" ]]; then
+  # If not, use latest.
+  SCANNER_DATA_VERSION="latest"
+elif [[ "$(wc -l <<< "${tag}")" -eq 1 ]]; then
+  # If there is exactly one tag on the commit, use that.
+  SCANNER_DATA_VERSION="${tag}"
+else
+  >&2 echo -e "Error: the HEAD commit has multiple tags, don't know which one to choose:\n${tag}"
+  exit 5
+fi
+
 for blob in "${blobs[@]}"; do
 
-  # TODO(ROX-22130): Assign proper suffix for tagged commits instead of /latest/.
-  url="https://storage.googleapis.com/definitions.stackrox.io/scanner-data/latest/${blob}"
+  url="https://storage.googleapis.com/definitions.stackrox.io/scanner-data/${SCANNER_DATA_VERSION}/${blob}"
   dest="${TARGET_DIR}/blob-${blob}"
 
-  echo "Downloading ${url} > ${dest}"
-  curl --fail -s --show-error --retry 4 --retry-max-time 30 --retry-connrefused \
+  echo "Downloading ${url} > ${dest}, retrying 1000 times or until killed..."
```

Comment:
Since we don't need to wait for `latest` blobs to appear, they are already there, I suggest assigning lower retry count to it. This is a small optimization but may give us quicker pipelines if things go wrong for most common `master` and PR builds.